src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntityChangeServiceTest.java

```diff
@@ -114,7 +114,7 @@ void testCollateOverseasEntityChangesUpdatedPrincipalAddressSameAsCorrespondence
         var result = overseasEntityChangeService.collateOverseasEntityChanges(existingRegistration, updateSubmission);
 
         assertEquals(1, result.size());
-        assertTrue(result.get(0) instanceof PrincipalAddressChange);
+        assertTrue(result.get(0) instanceof CorrespondenceAddressChange);
     }
 
     @Test
```

Comment:
Should fix the existing correspondence/principal address tests so that they match with service/regOff address respectively